qiskit_experiments/library/characterization/t1.py

```diff
@@ -43,7 +42,8 @@ class T1(BaseExperiment):
         by fitting to an exponential curve.
 
     # section: analysis_ref
-        :py:class:`T1Analysis`
+        :py:class:`.T1Analysis`
```

Comment:
What does the dot in the beginning mean?

Reply:
There is a description [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects) at the bottom of the section. Whether the dot is good here or not, I am not sure. T1Analysis is imported here so a search with the module name prepended should find it, but the real definition of T1Analysis is in a different module.

Reply:
I have not tested, but using the `.` when there is a reference in scope might help suppress warnings like this one that I see in the output from the docs build:

> `/home/runner/work/qiskit-experiments/qiskit-experiments/docs/tutorials/getting_started.rst:100: WARNING: more than one target found for cross-reference 'run': qiskit_experiments.calibration_management.BaseCalibrationExperiment.run, qiskit_experiments.curve_analysis.BaseCurveAnalysis.run, qiskit_experiments.curve_analysis.BlochTrajectoryAnalysis.run`

Reply:
I added this particular dot on the off chance it fixed all the T1 warnings (it did not), but it's optional here. In general, if the object name is unique, using dot then the name is much shorter than writing out the full path. A common problem in our docs are broken cross references because the wrong path is used or people tried to reference an object in a different path without the dot, so I think it's good to get into the habit of writing it this way. One caveat with the dot is when there are multiple objects with the same name, as shown in @wshanks's example, where you should use something like `` :meth:`.BaseCurveAnalysis.run` `` to disambiguate instead if it's not your local `run`.